client/me/billing-history/table-rows.js

```diff
@@ -1,21 +1,24 @@
 /**
  * External dependencies
  */
-var i18n = require( 'i18n-calypso' ),
-	some = require( 'lodash/some' ),
-	values = require( 'lodash/values' ),
-	isDate = require( 'lodash/isDate' ),
-	omit = require( 'lodash/omit' ),
-	flatten = require( 'lodash/flatten' );
+import { moment } from 'i18n-calypso';
+import {
+	flatten,
+	isDate,
+	omit,
+	some,
+	values,
+	without,
+} from 'lodash';
 
 function formatDate( date ) {
-	return i18n.moment( date ).format( 'MMM D, YYYY' );
+	return moment( date ).format( 'MMM D, YYYY' );
 }
 
 function getSearchableStrings( transaction ) {
-	var rootStrings = values( omit( transaction, 'items' ) ),
+	const rootStrings = values( omit( transaction, 'items' ) ),
 		transactionItems = transaction.items || [],
-		itemStrings = flatten( transactionItems.map( values ) );
+		itemStrings = without( flatten( transactionItems.map( values ) ), null, undefined );
 
 	return rootStrings.concat( itemStrings );
```

Comment:
rootStrings may also contain falsey values, so let's move the without to this line:
```
return without( rootStrings.concat( itemStrings ), null, undefined );
```